kratos/tests/test_normal_utils.py

```diff
@@ -43,6 +43,63 @@ def test_ComputeSimplexNormalModelPart(self):
             residual = math.sqrt((solution_normal[0]-normal[0])**2+(solution_normal[1]-normal[1])**2+(solution_normal[2]-normal[2])**2)
             self.assertLess(residual, 0.15)
 
+    def test_ComputeUnitNormalModelPart(self):
```

Comment:
this test must also be run in MPI and the results have to be the same, given that we recently had problems

@roigcarlo is this already #7178?

once you are done with the changes I can port it such that it works/runs both with and wthout mpi

Reply:
For MPI we will have to test particular corner cases, but agree with you, this should also run in MPI and give the same results. 

I am taking a look at the new implementation in case I see something odd with mpi.

Reply:
The tests we do in @KratosMultiphysics/altair follow this structure so they can be run both in mpi and openmp.

Reply:
didn't I suggest to do it like this in the first place :D

In any case I wanted for a while to unify sth like this in the core => separate PR (looking @sunethwarna for someone to help me with this :smiley:)

Reply:
we will get back to you asap, I think this PR could really benefit from it!

Reply:
@sunethwarna and I discussed, we will get back to you by the end of the week with sth that can run the test both in serial and in MPI